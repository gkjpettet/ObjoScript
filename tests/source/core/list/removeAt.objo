var list = ["a", "b", "c"]

list.removeAt(0)
System.print(list) // expect [b, c]
list = ["a", "b", "c"]

list.removeAt(1)
System.print(list) // expect [a, c]
list = ["a", "b", "c"]

list.removeAt(2)
System.print(list) // expect [a, b]
list = ["a", "b", "c"]

list.removeAt(-1)
System.print(list) // expect [a, b]
list = ["a", "b", "c"]

list.removeAt(-2)
System.print(list) // expect [a, c]
list = ["a", "b", "c"]

list.removeAt(-3)
System.print(list) // expect [b, c]
list = ["a", "b", "c"]