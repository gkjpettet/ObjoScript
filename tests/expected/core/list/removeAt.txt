[b, c]
[a, c]
[a, b]
[a, b]
[a, c]
[b, c]